queue_processors/queue_processor/tests/test_handle_message.py

```diff
@@ -9,492 +9,360 @@
 Tests message handling for the queue processor
 """
 
+from functools import partial
+import random
 import unittest
 from unittest import mock
-from unittest.mock import patch
+from unittest.mock import Mock, patch
 
-from model.database import access
+from django.db.utils import IntegrityError
+from parameterized import parameterized
```

Comment:
Is this a new dependency, it shows as not available for me and if so this will need to go into the `requirements.txt`

Reply:
It's added but you'll have to rerun `pip install -r requirements.txt`